etc/add-license.sh

```diff
@@ -1,15 +1,16 @@
 #!/bin/sh
 
-copyright=$'// Copyright (C) MongoDB, Inc. 2017-present.
+copyright=$'// Copyright (C) MongoDB, Inc. 2022-present.
```

Comment:
Optional: Consider getting the current year by running `date +"%Y"` instead of hard-coding it.

Reply:
That's a really good idea!